app/code/community/Oyst/OneClick/Model/Catalog.php

```diff
@@ -941,57 +941,36 @@ private function getCartRules(&$magentoQuoteBuilder, &$oneClickOrderCartEstimate
 
                 continue;
             }
+        }
 
-            // Manage classic rules
-            if (is_null($quoteAppliedRuleIds = $quoteItemData['applied_rule_ids'])) {
+        // Handle Classic discount rules
+        foreach($magentoQuoteBuilder->getQuote()->getTotals() as $total) {
```

Comment:
Add space after `foreach`.
You can use https://github.com/fabpot/PHP-CS-Fixer with PSR2 : `php-cs-fixer --rules=@PSR2 fix app/code/community/Oyst/OneClick/Model/Catalog.php`